### ISSUE TITLE
Columns section broken in generated project documentation (0.11.0rc2)

### ISSUE THREAD

**Issue by swiffer** [fishtown-analytics/dbt#975]
## Issue
Trying out 0.11.0rc2. Impressed by the effort around building documentation automatically. Migrated schema.yml to v2 and run the commands to built and serve docs. Sadly the Columns section seems to be broken.

### Issue description
- using Snowflake as Data Warehouse
- models and sql is lowercase in dbt

```sh
dbt docs generate
dbt docs serve
```

### Results
column duplicated (uppercase / lowercase):
![grafik](https://user-images.githubusercontent.com/2990373/45094444-96fa0080-b11b-11e8-89e3-13f460260879.png)

columns empty when there are more than one columns describet in schema.yml for one model:
![grafik](https://user-images.githubusercontent.com/2990373/45094572-007a0f00-b11c-11e8-8f7e-470729d494c4.png)

### System information
The output of `dbt --version`:
```
installed version: 0.11.0-rc2
   latest version: 0.10.2

Your version of dbt is ahead of the latest release!
```

The operating system you're running on:

The python version you're using (probably the output of `python --version`)

### Steps to reproduce
```yaml

version: 2

models:
  - name: fxl_styles
    description: A table containing styles

    columns:
      - name: style_guid
        description: This is a identifier for the style
        tests:
          - not_null
          - unique
          - relationships: { to: ref('fxl_items'), field: guid }

  - name: fxl_skus
    description: "A table containing skus (style + color + size variants)"

    columns:
      - name: style_guid
        description: This is a identifier for the style
        tests:
          - not_null
          - relationships: { to: ref('fxl_items'), field: guid }

      - name: color_guid
        description: This is a identifier for the color
        tests:
          - not_null
          - relationships: { to: ref('fxl_colors'), field: guid }
```

**Comment by drewbanin** [comment]
Thanks for the report @swiffer!

I think what's happening here is that dbt gets column information from two different sources:
1) The schema.yml files
2) The `information_schema.columns` table

The docs site then merges these two lists of columns to provide a column type (from the database) and a description (from schema.yml). Since Snowflake auto-uppercases identifiers, it looks like dbt isn't correctly merging the two together.

I guess there are two options here:
1) dbt could do a case-insensitive match on column name (eg. `style_guid == STYLE_GUID`)
2) you could uppercase your column names in `schema.yml` to facilitate the match

The second option above would be pretty annoying in practice, so I'm definitely interested in implementing the first one.

Separately, can you elaborate on:
> columns empty when there are more than one columns described in schema.yml for one model

If you have an example schema.yml, that would be really helpful!

**Comment by swiffer** [comment]
Thanks @drewbanin responding that fast!

if it does not trigger other errors beeing case insensitive would be appreciated 👍 

yes, i've added this to the bottom (see model fxl_skus)

**Comment by drewbanin** [comment]
Ok - I think I have a good patch for the column insensitive match. The 0.11.0 release is scheduled to go out tomorrow, so I'm unsure if I'll be able to sneak it in for 0.11.0. At the very latest, it will go out in an 0.11.1 bugfix patch!

I can't reproduce the empty column section issue you're seeing! Do you see any warnings in the `logs/dbt.log` file? My guess is that there's an issue with the schema.yml entry and dbt is ignoring that whole model.

**Comment by swiffer** [comment]
no, nothing special in there - running `dbt test` also executes tests for all models specified in schema.yml
let's see after patch is applied ;)

**Comment by swiffer** [comment]
insensitive match is working fine in 0.11.0
columns section is populated only when the model is defined in schema.yml, right?

**Comment by swiffer** [comment]
however the test section is not shown

![grafik](https://user-images.githubusercontent.com/2990373/45207990-ce86bb00-b289-11e8-94a8-8ed168b26981.png)

schema.yml
```yaml
    columns:
      - name: sku_guid
        description: This is a identifier for the sku
        tests:
          - not_null
          - unique
          - relationships: { to: ref('fxl_items'), field: guid }
```

**Comment by drewbanin** [comment]
ok, thanks @swiffer. We'll fix this in the 0.11.1 patch release. Thanks for the heads up!

**Comment by drewbanin** [comment]
@swiffer this is fixed over here: https://github.com/fishtown-analytics/dbt-docs/pull/2
